genesyscloud/resource_genesyscloud_recording_media_retention_policy.go

```diff
@@ -0,0 +1,2715 @@
+			"evaluation_context_id": {
+				Description: "",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"evaluators": {
+				Description: "",
+				Type:        schema.TypeList,
+				Optional:    true,
+				Elem:        user,
+			},
+			"max_number_evaluations": {
+				Description: "",
+				Type:        schema.TypeInt,
+				Optional:    true,
+			},
+			"evaluation_form_id": {
+				Description: "",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"evaluation_form_context_id": {
+				Description: "",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"assign_to_active_user": {
+				Description: "",
+				Type:        schema.TypeBool,
+				Optional:    true,
+			},
+			"time_interval": {
+				Description: "",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Elem:        timeInterval,
+			},
+		},
+	}
+
+	meteredAssignmentByAgent = &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"evaluation_context_id": {
+				Description: "",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"evaluators": {
+				Description: "",
+				Type:        schema.TypeList,
+				Optional:    true,
+				Elem:        user,
+			},
+			"max_number_evaluations": {
+				Description: "",
+				Type:        schema.TypeInt,
+				Optional:    true,
+			},
+			"evaluation_form_id": {
+				Description: "",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"evaluation_form_context_id": {
+				Description: "",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"time_interval": {
+				Description: "",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Elem:        timeInterval,
+			},
+			"time_zone": {
+				Description: "",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+		},
+	}
+
+	calibrationAssignment = &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"calibrator_id": {
+				Description: "",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"evaluators": {
+				Description: "",
+				Type:        schema.TypeList,
+				Optional:    true,
+				Elem:        user,
+			},
+			"evaluation_form_id": {
+				Description: "",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"evaluation_form_context_id": {
+				Description: "",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"expert_evaluator_id": {
+				Description: "",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+		},
+	}
+
+	surveyAssignment = &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"survey_form": {
+				Description: "The survey form used for this survey.",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Elem:        publishedSurveyFormReference,
+			},
+			"flow_id": {
+				Description: "The URI reference to the flow associated with this survey.",
```

Comment:
Probably best to change the description here. This would be a UUID reference because we're not using the domainEntityRef.